cmd/protoc-gen-go-errors/errors.go

```diff
@@ -82,21 +82,43 @@ func genErrorsReason(gen *protogen.Plugin, file *protogen.File, g *protogen.Gene
 		if enumCode == 0 {
 			continue
 		}
+
+		comment := v.Comments.Leading.String()
+		if comment == "" {
+			comment = v.Comments.Trailing.String()
+		}
+
+		camelValue := case2Camel(string(v.Desc.Name()))
+		comment = buildComment(camelValue, comment)
+
 		err := &errorInfo{
 			Name:       string(enum.Desc.Name()),
 			Value:      string(v.Desc.Name()),
 			CamelValue: case2Camel(string(v.Desc.Name())),
 			HTTPCode:   enumCode,
+			Comment:    comment,
+			HasComment: len(comment) > 0,
 		}
 		ew.Errors = append(ew.Errors, err)
 	}
 	if len(ew.Errors) == 0 {
 		return true
 	}
 	g.P(ew.execute())
+
 	return false
 }
 
+// buildComment returns comment content with prefix //
+func buildComment(camelValue, comment string) string {
+	if comment == "" {
+		return ""
+	}
+
+	comment = strings.Replace(comment, "//", "", 1)
+	return fmt.Sprintf("// %s %s", camelValue, comment)
```

Comment:
I think this is an implicit behavior, the user should be responsible for his own Comment, not we generate a camelValue for it